setup.py

```diff
@@ -25,9 +25,13 @@ def readme():
           'pyhocon',
           'mock',
           'arrow',
-          'nose'
+          'nose',
+          'requests',
+          'thrift==1.0.0-py3',
+          'munch'
       ],
       entry_points={'console_scripts': ['herc = herc.webservice:main']},
+      dependency_links = [urllib.parse.urljoin('file:', os.path.join(os.getcwd(), 'thrift-1.0.0-py3.tar.gz'))],
```

Comment:
In the event that this project gets pushed to PyPI, it might only upload data explicitly referenced in the setup.py. But honestly I have no idea. I do know it works off a fresh `git clone`, so we could potentially just punt the issue for now.